packages/block-editor/src/components/block-list/style.scss

```diff
@@ -559,33 +557,16 @@
 		}
 	}
 
-	// Full-wide
+	// Full-wide.
 	&[data-align="full"] {
 		// Position hover label on the left for the top level block.
 		> .block-editor-block-list__block-edit > .block-editor-block-list__breadcrumb {
 			left: 0;
 		}
 
-		// Compensate for main container padding and subtract border.
-		@include break-small() {
-			margin-left: -$block-side-ui-width - $block-padding - $block-side-ui-clearance - $border-width;
-			margin-right: -$block-side-ui-width - $block-padding - $block-side-ui-clearance - $border-width;
-		}
-
 		> .block-editor-block-list__block-edit {
-			margin-left: -$block-padding;
-			margin-right: -$block-padding;
-
-			@include break-small() {
-				margin-left: -$block-side-ui-width - $block-side-ui-clearance - $block-padding;
-				margin-right: -$block-side-ui-width - $block-side-ui-clearance - $block-padding;
-			}
-
-			// This explicitly sets the width of the block, to override
-			// `width: fit-content` from the image block.
-			figure {
-				width: 100%;
-			}
+			margin-left: -$block-padding - $block-padding - $block-side-ui-width - $border-width - $border-width;
+			margin-right: -$block-padding - $block-padding - $block-side-ui-width - $border-width - $border-width;
```

Comment:
I think there may be a styling issue introduced by these changes.

1. Create a Table block
2. Change its alignment to full width
3. Check mobile screen

![image](https://user-images.githubusercontent.com/3068563/68673677-97c6c780-0554-11ea-8a31-836c9d8c6e84.png)